ModernFlyouts/App.xaml.cs

```diff
@@ -14,13 +10,11 @@ public App()
             Startup += App_Startup;
             JumpListHelper.CreateJumpList();
         }
-        
+
         private void App_Startup(object sender, StartupEventArgs e)
         {
             FlyoutHandler.Instance = new FlyoutHandler();
             FlyoutHandler.Instance.Initialize();
-            AppCenter.Start("26393d67-ab03-4e26-a6db-aa76bf989c21",
-                typeof(Analytics), typeof(Crashes));
         }
     }
-}
+}
```

Comment:
Why are line endings removed everywhere?

It's a convention to keep a empty line at the end of a code file, isn't it?

Could this be reverted?